wiki/Disambiguation/Off-topic/en.md

```diff
@@ -0,0 +1,12 @@
+---
+tags:
+  - offtopic
+  - OT
+---
+
+# Off-topic (disambiguation)
+
+**Off-topic** can have multiple meanings:
+
+- [Chat](/wiki/Internet_Relay_Chat) or [forum](/wiki/Community/Forum) messages that don't follow the ongoing discussion's theme.
```

Comment:
I... don't know if this point justifies this whole disambiguation page.

Reply:
To explain a little further, no other disambiguation page is made purely because an osu term can be confused for the literal meaning of the word (may be a stretch for this case as well). They all have in common that a term has multiple meanings related to osu. Being "off-topic" isn't only related to osu chat or forum messages.

By this logic, "Tournaments" should have it's own disambiguation page for being confused with "the Tournaments forum" and "tournaments in osu (specifically)"

Reply:
fair, dunno why I've got that idea